coalib/collecting/Importers.py

```diff
@@ -28,6 +25,18 @@ def _import_module(file_path):
                 module_name = cased_module_name
                 break
 
+    if module_dir not in sys.path:
```

Comment:
`module_dir` should never be in `sys.path`.  If it is, that is a bug.

Reply:
ok, so I think we need to remove this `if` block and assuming that `module_dir` would never be in `sys.path`?

Reply:
`module_dir` should *never* be in `sys.path`.

Add an assert that `module_dir` is not in `sys.path`.

Destroy this `if` block, and rewrite this patch based on the above, fixing any test which violates this rule.